src/main/steps/applicant1/apply-for-help-with-fees/content.ts

```diff
@@ -1,25 +1,21 @@
 import { TranslationFn } from '../../../app/controller/GetController';
 import { HELP_WITH_YOUR_FEE_URL } from '../../urls';
 
-const en = ({ isDivorce, isJointApplication, partner }) => ({
+const en = ({ isDivorce }) => ({
   title: `You need to apply for help with your ${isDivorce ? 'divorce' : ''} fees`,
-  line1: `Your need to apply for <a href="https://www.gov.uk/get-help-with-court-fees" class="govuk-link" target="_blank">help with your fees (opens in new tab)</a> before you continue with this ${
+  line1: `Your need to <a href="https://www.gov.uk/get-help-with-court-fees" class="govuk-link" target="_blank">apply for help with your fees (opens in new tab)</a> before you continue with this ${
```

Comment:
should this be 'You need to'?

Reply:
im going to assume youre right as it makes sense